src/test/java/com/BlogSite/TestBlogProject/services/UserServiceImplTest.java

```diff
@@ -24,71 +22,98 @@ class UserServiceImplTest {
     @Mock
     private UserRepository userRepository;
     @InjectMocks
-    private UserServiceImpl test;
+    private UserServiceImpl userService;
 
     @Test
     void getUsers_ShouldReturnAllUsers() {
```

Comment:
this is a good example of useless test.
You have checked that repo method was called but it means completely nothing.
What if your getUsers() implementation will looks like:

`public List<User> getUsers(){
      userRepository.findAll();
      return Collections.emptyList();
}`